### ISSUE TITLE
Running Program error

### ISSUE THREAD

**Issue by viking2** [RealRaven2000/FiltaQuilla#254]
I am still trying to figure out how to get FiltaQuilla to show a notification window (see here: https://github.com/RealRaven2000/FiltaQuilla/issues/240).

I found a native Windows program that can send one message at a time; For example:
`C:\Windows\System32\msg.exe peter "hello"` , where peter is my windows username, 
would open a window with text "hello". It works fine from the command prompt. 

However, when I run it as a program in FiltaQuilla, I get an error:
`Applying filter Copy of test failed. Would you like to continue applying filters? `

p.s. running 
`C:\Windows\System32\notepad.exe ` works fine

**Comment by RealRaven2000** [comment]
> I am still trying to figure out how to get FiltaQuilla to show a notification window (see here: #240).
> 
> I found a native Windows program that can send one message at a time; For example: `C:\Windows\System32\msg.exe peter "hello"` , where peter is my windows username, would open a window with text "hello". It works fine from the command prompt.
> 
> However, when I run it as a program in FiltaQuilla, I get an error: `Applying filter Copy of test failed. Would you like to continue applying filters? `
> 
> p.s. running `C:\Windows\System32\notepad.exe ` works fine

Can you make a error console log? If there are no exceptions there, it may happen lower down at the filter service layer (which we Add-on developers have no access to) - the error message looks like it comes from there.

How to create the JavaScript log is described here:

https://quickfilters.quickfolders.org/bugs.html#createlog

(you can ignore the section on extra log detail by right-clicking debug, although this technique also works in FIltaQuilla - these are special debug switches that add detail, but I don't think it's useful here).

**Comment by viking2** [comment]
Here is what I get after editing the filter
```
FiltaQuilla [logTime init]
 fq_FilterEditor.js - start... [filtaquilla-util.js:282:13](chrome://filtaquilla/content/filtaquilla-util.js)
FiltaQuilla 10:47:40.626  [1 ms]   
 fq_FilterEditor.js - Finished. [filtaquilla-util.js:282:13](chrome://filtaquilla/content/filtaquilla-util.js)
searchTerm: termIdxFromRichlistIdx 0 -> -1 [searchTerm.js:58:15](chrome://messenger/content/searchTerm.js)
searchTerm: termIdxFromRichlistIdx 1 -> 0 [searchTerm.js:70:13](chrome://messenger/content/searchTerm.js)
searchTerm: initializeSearchRows - exit 0: and   level 1 [searchTerm.js:44:13](chrome://messenger/content/searchTerm.js)
[MailboxAlert] prevMethod: function(type) {
          let element = getChildNode(type);
          return element ? element : prevMethod(type);
        }
TypeError: gFilter.searchTerms[i] is undefined
[qFilters-filterEditor.js:142:15](chrome://quickfilters/content/qFilters-filterEditor.js)
```
After I try to run the filter, I get:
```
Uncaught 
Exception { name: "NS_ERROR_FILE_INVALID_PATH", message: "Component returned failure code: 0x80520009 (NS_ERROR_FILE_INVALID_PATH) [nsIProcess.init]", result: 2152857609, filename: "chrome://filtaquilla/content/filtaquilla.js", lineNumber: 564, columnNumber: 0, data: null, stack: "applyAction@chrome://filtaquilla/content/filtaquilla.js:564:22\n", location: XPCWrappedNative_NoHelper }
```


**EDIT:** I figured out the issue; at the command prompt  I need to use the syntax:
`C:\Windows\System32\msg.exe peter "hello"`

However, in FiltaQuilla I need to add commas:
`C:\Windows\System32\msg.exe, peter, "hello"`

I would like the message box to display the subject etc, so I tried this:
`C:\Windows\System32\msg.exe, peter, @SUBJECT@, @AUTHOR@`
(p.s. I first tried @subject@ and that didn't work.....)

**That worked :-)**! 
I get the output in the message box, showing the E-mail subject and author!

The disadvantages with this approach is that:
1.  I can only see one message at a time. I need to click on "OK" in the message box to see the next one.
2. I can not click on the message box to take me to the E-mail.

**Comment by OlehRA** [comment]
thanks, good job! 
But i have problems with cyrilic symbols in a subject. ) 
It's a next disadvantage with this approach. But i like it.

**Comment by RealRaven2000** [comment]
> thanks, good job! But i have problems with cyrilic symbols in a subject. ) It's a next disadvantage with this approach. But i like it.

Can you please add a separate bug, maybe we can decode the subject properly...

**Comment by OlehRA** [comment]
There is shown the name of sender and a subject. The subject is "Hello! Привіт!".  
![2024_04_25_09_29_02_Clipboard](https://github.com/RealRaven2000/FiltaQuilla/assets/115365160/a8511097-457c-46b9-a996-90f488c906d2)

**Comment by RealRaven2000** [comment]
> There is shown the name of sender and a subject. The subject is "Hello! Привіт!". 

right- but please don't piggy back new issues on existing bugs - make new ones so we can deal with them. This has nothing to do with the original issue, and I cannot close #254 that way.
